src/subscribables/dependentObservable.js

```diff
@@ -52,6 +51,29 @@ ko.computed = ko.dependentObservable = function (evaluatorFunctionOrOptions, eva
         }
     }
 
+    // evaluatePossiblyAsync is defined in the constructor so that it can access the computed properties
+    // and method using a closure instead of "this", meaning that we can pass the function to "subscribe"
+    // without the need to use "bind". Older versions of Firefox seem to use up a lot more stack space
+    // when using a bound function or using "call". See #1622.
+    var evaluationTimeoutInstance = null;
+    computedObservable.evaluatePossiblyAsync = function () {
```

Comment:
If we're okay with the test failing in older FF, then we can simply ignore these changes and possibly skip the test for those browsers. After all, the test already failed with version 3.3.0.